rvic/parameters.py

```diff
@@ -319,14 +335,13 @@ def gen_uh_run(uh_box, fdr_data, fdr_vatts, dom_data, outlet, config_dict,
     """
```

Comment:
This is just a general comment - maybe you can add more detailed descriptions for each function about what it is doing and what its input/output variables and their type/structure are? For each piece of code - maybe add also add more comments about their purpose? For major variables - maybe add their data type/dimension/etc.?